Assignment 3/responsive.css

```diff
@@ -0,0 +1,121 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
+  }
+  
+  header {
+    background: #333;
+    color: #fff;
+    padding: 20px;
+    text-align: center;
+  }
+  
+  nav ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  nav ul li {
+    display: inline;
+    margin-right: 10px;
+  }
+  
+  nav ul li a {
+    color: #fff;
+    text-decoration: none;
+  }
+  
+  .intro {
+    text-align: center;
+    padding: 40px;
+    background: #f4f4f4;
+  }
+  
+  .features {
+    display: flex;
+    justify-content: space-around;
+    padding: 20px;
+  }
+  
+  .feature-item {
+    background: #fff;
+    padding: 20px;
+    border: 1px solid #ddd;
+    width: 30%;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  footer {
+    text-align: center;
+    padding: 20px;
+    background: #333;
+    color: #fff;
+  }
+  
+  @media ( max-width: 475px) {
+    .features{
+      display: flex;
+      flex-direction: column;
+    }
+    .feature-item{
+      display: flex;
+      flex-direction: row;
+      flex-wrap: nowrap;
+      max-width: 640px;
+      height: 30%;
+    }
+    
+    }
+  
+  @media ( max-width: 640px) {
+    .features{
+      display: flex;
+      flex-direction: column;
+      width: 310%;
+  
+    }
+    .feature-item{
+      display: flex;
+      flex-direction: column;
+      max-width: 640px;
+      height: 30%;
+      align-content: space-between;
+    }
+  }
+  
+  @media ( max-width: 768px) {
+    .features{
+      display: flex;
+      flex-direction: column;
+      width: 310%;
+  
+    }
+    .feature-item{
+      display: flex;
+      flex-direction: column;
+      max-width: 640px;
+      height: 30%;
+      align-content: space-between;
+    }
+  }  
+  
+  
+  @media ( max-width: 1024px) {
+    .feature-item {
+      background-color: green;
+    }
+  }  
+  
+  
+  @media ( max-width: 1280px) {
+    .feature-item {
+    background-color: #fff;
+    }
+  }  
```

Comment:
Oppgaven sa 5 størrelser mindre enn desktop, men disse er desktop og tv size (large og xl i tailwind https://tailwindcss.com/docs/responsive-design)

Reply:
Jeg vet egentlig ikke hva han leter etter for flere mindre skjermstørrelser da, det er veldig rart å ha flere breakpoints